recipes/mtspec/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "307dc34acefdfae06f055d99d0260cb3529a1fd46d7696be2ab533752aff01a1" %}
+
+package:
+  name: mtspec
+  version: {{ version }}
+
+
+source:
+  fn: mtspec-{{ version }}.zip
+  url: https://files.pythonhosted.org/packages/e8/23/a6b16605dbe1e47a2a6128957ffd4c6d3aa06bd2971de4de3825e1f6f65d/mtspec-0.3.0.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+    - numpy
+  run:
+    - python
+    - libgcc
+    - numpy
+
+test:
+  imports:
+    - mtspec
+  commands:
+    - python -m mtspec.tests
+
+about:
+  home: http://github.com/krischer/mtspec
+  license: GPL-3.0
+  summary: 'Python library for multitaper spectral estimations'
+  description: |
+    mtspec is a Python (ctypes) wrapper for the Multitaper Spectrum Estimation
+    Library mwlib.a by Germán A. Prieto.
```

Comment:
The linter is probably choking on the unicode letter `á`.  And I thought we fixed that :unamused: